packages/sdk/src/actions/signUpdateDeposit.ts

```diff
@@ -5,7 +5,7 @@ import { simulateUpdateDepositTx } from "./simulateUpdateDepositTx.js";
 
 export type SignUpdateDepositTypedDataParams = {
   walletClient: WalletClient;
-  depositId: number;
+  depositId: bigint;
```

Comment:
So this is a breaking change right? Maybe it makes sense to allow `number  | bigint` here and convert to a bigint in the function logic

Reply:
Yeah you're right, I think we can change this to a union of bigint and number and just cast is to bigint in the implementation.